catt/cli.py

```diff
@@ -362,14 +362,16 @@ def save(settings, path):
     if path and path.is_file():
         click.confirm("File already exists. Overwrite?", abort=True)
     click.echo("Saving...")
-    state = CastState(path or STATE_PATH, create_dir=True if not path else False)
+    state = CastState(path or STATE_PATH, mode="arbi" if path else "conf")
```

Comment:
How this is going to be parsed is pretty confusing, it needs parentheses to clarify.

Reply:
Around both parts, or just the latter?

Reply:
Oh, I misread this due to the complexity. I would just split it up:

```
if path:
    state = CastState(path, mode.ARBI)
else:
    state = CastState(STATE_PATH, mode.CONF)
```